Modules/Sources/WordPressUI/Views/Users/Views/UserDetailView.swift

```diff
@@ -0,0 +1,178 @@
+
+    @StateObject
+    var viewModel: UserDetailViewModel
+
+    @Environment(\.dismiss)
+    var dismissAction: DismissAction
+
+    init(user: DisplayUser, userProvider: UserDataProvider, actionDispatcher: UserManagementActionDispatcher) {
+        self.user = user
+        self.userProvider = userProvider
+        self.actionDispatcher = actionDispatcher
+        _viewModel = StateObject(wrappedValue: UserDetailViewModel(userProvider: userProvider))
+    }
+
+    var body: some View {
+        Form {
+            Section(Strings.nameSectionTitle) {
+                LabeledContent(Strings.roleFieldTitle, value: user.role)
+                LabeledContent(Strings.firstNameFieldTitle, value: user.firstName)
+                LabeledContent(Strings.lastNameFieldTitle, value: user.lastName)
+                LabeledContent(Strings.nicknameFieldTitle, value: user.handle)
+                LabeledContent(Strings.displayNameFieldTitle, value: user.displayName)
+            }
+
+            Section(Strings.contactInfoSectionTitle) {
+                LabeledContent(Strings.emailAddressFieldTitle, value: user.emailAddress)
+                if let website = user.websiteUrl {
+                    LabeledContent(Strings.websiteFieldTitle, value: website)
+                }
+            }
+
+            Section(Strings.aboutUserSectionTitle) {
+                LabeledContent(Strings.bioFieldTitle, value: user.biography ?? "")
+                if let profilePhotoUrl = user.profilePhotoUrl {
+                    LabeledContent(Strings.profilePictureFieldTitle) {
+                        UserProfileImage(size: CGSize(width: 96, height: 96), url: profilePhotoUrl)
+                    }
+                }
+            }
+
+            if viewModel.currentUserCanModifyUsers {
+                Section(Strings.accountManagementSectionTitle) {
+                    NavigationLink {
+                        UserChangePasswordView(viewModel: passwordChangeViewModel)
+                    } label: {
+                        Text(Strings.setNewPasswordActionTitle)
+                    }
+
+                    NavigationLink {
+                        // Pass this view's dismiss action, because if we delete a user, we want that screen *and* this one gone
+                        UserDeleteView(user: user, userProvider: userProvider, actionDispatcher: actionDispatcher, dismiss: dismissAction)
+                    } label: {
+                        Text(Strings.deleteUserActionTitle)
+                    }
+                }
+            }
+        }
+        .navigationTitle(user.displayName)
+        .task {
+            await viewModel.loadCurrentUserRole()
+        }
+    }
+
+    var passwordChangeViewModel: UserChangePasswordViewModel {
+        UserChangePasswordViewModel(user: user, actionDispatcher: actionDispatcher)
+    }
+
+    enum Strings {
+        static let nameSectionTitle = NSLocalizedString(
+            "userdetail.name-section-title",
```

Comment:
These need to use cameCase https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys: `userDetails.nameSectionTitle".

(nit) I'd also suggesting keeping the localizable strings as one liners. You don't really need to make them readable and they take too much vertical space otherwise.